DatadogSessionReplay/Tests/Mocks/SRDataModelsMocks.swift

```diff
@@ -240,8 +240,8 @@ extension SRShapeStyle: AnyMockable, RandomMockable {
     public static func mockRandom() -> SRShapeStyle {
         return SRShapeStyle(
             backgroundColor: .mockRandom(),
-            cornerRadius: .mockRandom(),
-            opacity: .mockRandom()
+            cornerRadius: .randomInteger(),
+            opacity: .randomInteger()
```

Comment:
We no longer mock random `Double` as it was giving different results when serializing high-precission numbers with `Codable` vs `JSONSerialization` on iOS 17. In some tests we compare JSON strings, so to avoid flakiness now we only randomize these values among integer numbers from a realistic range (not the full `Double` range).